src/org/labkey/test/pages/study/specimen/ManageNotificationsPage.java

```diff
@@ -0,0 +1,175 @@
+package org.labkey.test.pages.study.specimen;
+
+import org.jetbrains.annotations.Nullable;
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Checkbox;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.RadioButton;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.List;
+
+public class ManageNotificationsPage extends LabKeyPage<ManageNotificationsPage.ElementCache>
```

Comment:
Aren't we adding a comment at the top identifying the dev code / page object this represents?